web-common/src/AjaxUtils.purs

```diff
@@ -58,28 +56,15 @@ ajaxErrorRefLabel = RefLabel "ajax-error"
 ajaxErrorClass :: ClassName
 ajaxErrorClass = ClassName "ajax-error"
 
-helpText :: forall p i. HTML p i
-helpText = text "Please try again or contact support for assistance."
-
 showAjaxError :: forall p i. AjaxError -> HTML p i
 showAjaxError = runAjaxError >>> _.description >>> showErrorDescription
 
 showErrorDescription :: forall p i. ErrorDescription -> HTML p i
-showErrorDescription NotFound = div_ [ text $ "Data not found." ]
+showErrorDescription NotFound = text "Data not found."
```

Comment:
Are you sure that everywhere we call `showErrorDescription`  we're not expecting it to be in a div? Marlowe too?

Reply:
Well if these changes are going to make something look weird, then the other instances that just return `text` instead of a `div` must already be looking weird. But you make a good point about Marlowe - so I checked with David and Hernan, and they're happy with the change. So I think we're good.

Reply:
Okay. Thanks for checking.